ts/src/huobi.ts

```diff
@@ -1699,6 +1699,13 @@ export default class huobi extends Exchange {
             // 7 Settlement Completed
             // 8 Delivered
             // 9 Suspending of Trade
+            let created = undefined;
+            let createdDate = this.safeString (market, 'create_date'); // i.e 20230101
+            if (createdDate !== undefined) {
+                const createdArray = this.stringToCharsArray (createdDate);
+                createdDate = createdArray[0] + createdArray[1] + createdArray[2] + createdArray[3] + '-' + createdArray[4] + createdArray[5] + '-' + createdArray[6] + createdArray[7] + ' 00:00:00';
+                created = this.parse8601 (createdDate);
```

Comment:
same, not consistent

Reply:
was this an issue? maybe you didn't mean here, because `parse8601` returns number, not string

Reply:
sorry, yes didn't mean that, but second question do we really need this custom stringToCharsArray function? We have for instance `slice`

Reply:
but `slice` with empty separator works only in JS and not transpilable, so in other langs we needed these base functions. if you can think of any other approach, lmk